app/models/annotations/dynamic.rb

```diff
@@ -19,15 +19,8 @@ class Dynamic < ActiveRecord::Base
 
   def slack_message
     if !self.set_fields.blank? && self.annotation_type =~ /^task_response/
-      response = note = task = '-'
-
-      @fields ||= self.fields
```

Comment:
I removed it because its purpose was not clear - plz mark performance optimizations with a comment. I will bring it back.